dom/frag/frag.js

```diff
@@ -24,6 +24,8 @@ Convert a String, HTMLElement, documentFragment, or contentArray into a document
 
 ContentArrays can be used to combine multiple HTMLElements into a single document fragment.  For example:
 
+    var frag = require("can-util/dom/frag/");
```

Comment:
We avoid the short (StealJS-friendly) path for users of other module loaders which don’t support this syntax.